app/views/guide/show.html.erb

```diff
@@ -0,0 +1,88 @@
+<% add_view_stylesheet("guide") %>
+
+<% content_for :extra_headers do %>
+  <meta name="description" content="<%= content_item.description %>">
+  <%= render "govuk_publishing_components/components/machine_readable_metadata", {
+    content_item: content_item.to_h,
+    schema: :article,
+    canonical_url: content_item.current_part_path,
+    body: content_item.current_part_body.presence,
+  } %>
```

Comment:
The `og:url` meta tag seems to have a relative path when it should be absolute on https://govuk-frontend-app-pr-5028.herokuapp.com/prove-right-to-rent